src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -210,7 +210,7 @@ static void PrintSettings(IEnumerable<KeyValuePair<string, object>> settings)
                             }
                         }
 
-                        await Task.Delay(1).ConfigureAwait(false);
+                        await Task.Yield();
```

Comment:
I think Task.Delay uses IO completion ports while I assume Task.Yield doesn't. In case the CPU resources on the build agent are really sparse, I could imagine that the IO completion ports LIFO nature might cause issues? This is highly speculative though. I wish we would now a little bit more about this, but if it's able to resolve the blocking build agents I guess that would be good enough?